corim/signedcorim.go

```diff
@@ -0,0 +1,144 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package corim
+
+import (
+	"crypto"
+	"crypto/rand"
+	"errors"
+	"fmt"
+
+	cose "github.com/veraison/go-cose"
+)
+
+var (
+	ContentType = "application/rim+cbor"
+)
+
+// SignedCorim encodes a signed-corim type (i.e., a COSE Sign1 wrapped CoRIM)
```

Comment:
Kindly re-word it slightly better to align it with the structure naming:
SignedCorim encodes a signed-corim message (.i.e, a COSE Sign1 wrapped CoRIM) with signature and verification methods.

Reply:
```suggestion
// SignedCorim encodes a signed-corim message (i.e., a COSE Sign1 wrapped CoRIM)
```

Reply:
sure (BTW, for this kind of editorial stuff you can suggest the correction directly on the review)